projects/aca-content/src/lib/components/toolbar/toggle-edit-offline/toggle-edit-offline.component.ts

```diff
@@ -117,13 +118,15 @@ export class ToggleEditOfflineComponent implements OnInit {
   }
 
   lockNode(nodeId: string) {
+    this.nodeActionsService.setNodeLocked(true);
```

Comment:
you should all this only when "nodesApi.lockNode" is complete

Reply:
if you need extra bug fixes after merging this PR then please add those bug fixes here, as the PR is not a clean implementation then

Reply:
All bugs in the new design have been fixed